### PR TITLE
dev/reference, v3.0/reference, v2.1/reference: add safe-mode config in binlog reparo doc

### PR DESCRIPTION
**Pull request by ericsyh**
### What is changed, added or deleted? 

In https://github.com/pingcap/tidb-binlog/pull/652 we add a new config `safe-mode` in reparo, this PR this to update the reparo documents about the new config. 

### What is the related PR or file link(s)? 

In https://github.com/pingcap/tidb-binlog/pull/652. 

### Which version does your change affect? 

Apply in dev/v3.0/v2.1

### CODE REVIEW

**Comment by yikeke** [comment]
**Important Update**: We have added a markdownlint static check in the CI check to improve the quality of our documentation, so your unclosed PR might fail the CI check now.

Click `details` beside the `ci/circleci: lint — Your tests failed on CircleCI` message, and you can check the details for the failure and fix all the issues in your PR. It is recommended to install "markdownlint" extension in your VS Code editor, so it will check all potential issues that fail the CI check. 

If you have any problem with troubleshooting, please consult @yikeke for help. Thanks for your cooperation~

Ref: https://github.com/pingcap/docs-cn/pull/1494

**Comment by WangXiangUSTC** [comment]
LGTM

**Comment by yikeke** [comment]
Please remove the hard tab in the Line 36 of the changed `dev/reference/tools/tidb-binlog/reparo.md` file, so your pull request can pass the CI check. Thanks~ @ericsyh

**Comment by ericsyh** [comment]
Okay, i will fix it.

File: dev/reference/tools/tidb-binlog/reparo.md [hunk 1]
```diff
@@ -32,6 +32,9 @@ Usage of Reparo:
     用于指定开始恢复的时间点，格式为 “2006-01-02 15:04:05”。如果不设置该参数则从最早的 binlog 文件开始恢复。
 -stop-datetime string
     用于指定结束恢复的时间点，格式同上。如果不设置该参数则恢复到最后一个 binlog 文件。
+-safe-mode bool
+    指定是否开启 safe mode，开启后可支持反复同步。
```
Comment (yikeke) [inline dev/reference/tools/tidb-binlog/reparo.md]:
```suggestion
    指定是否开启安全模式，开启后可支持反复同步。
```

File: dev/reference/tools/tidb-binlog/reparo.md [hunk 2]
```diff
@@ -59,6 +62,9 @@ log-level = "info"
 # 下游服务类型。 取值为 print, mysql（默认值：print）。当值为 print 时，只做解析打印到标准输出，不执行 SQL；如果为 mysql，则需要在 [dest-db] 中配置 host、port、user、password 等信息。
 dest-type = "mysql"
 
+# 安全模式配置。取值为 true, false (默认值：false)。当值为 true 时，Reparo 会将 update 语句拆分为 delete + replace 语句。
```
Comment (yikeke) [inline dev/reference/tools/tidb-binlog/reparo.md]:
```suggestion
# 安全模式配置。取值为 true 或 false（默认值：false）。当值为 true 时，Reparo 会将 update 语句拆分为 delete + replace 语句。
```

File: v2.1/reference/tools/tidb-binlog/reparo.md [hunk 3]
```diff
@@ -32,6 +32,9 @@ Usage of Reparo:
     用于指定开始恢复的时间点，格式为 “2006-01-02 15:04:05”。如果不设置该参数则从最早的 binlog 文件开始恢复。
 -stop-datetime string
     用于指定结束恢复的时间点，格式同上。如果不设置该参数则恢复到最后一个 binlog 文件。
+-safe-mode bool
+    指定是否开启 safe mode，开启后可支持反复同步。
```
Comment (yikeke) [inline v2.1/reference/tools/tidb-binlog/reparo.md]:
```suggestion
    指定是否开启安全模式，开启后可支持反复同步。
```

File: v2.1/reference/tools/tidb-binlog/reparo.md [hunk 4]
```diff
@@ -59,6 +62,9 @@ log-level = "info"
 # 下游服务类型。 取值为 print, mysql（默认值：print）。当值为 print 时，只做解析打印到标准输出，不执行 SQL；如果为 mysql，则需要在 [dest-db] 中配置 host、port、user、password 等信息。
 dest-type = "mysql"
 
+# 安全模式配置。取值为 true, false (默认值：false)。当值为 true 时，Reparo 会将 update 语句拆分为 delete + replace 语句。
```
Comment (yikeke) [inline v2.1/reference/tools/tidb-binlog/reparo.md]:
```suggestion
# 安全模式配置。取值为 true 或 false（默认值：false）。当值为 true 时，Reparo 会将 update 语句拆分为 delete + replace 语句。
```

File: v3.0/reference/tools/tidb-binlog/reparo.md [hunk 5]
```diff
@@ -33,6 +33,9 @@ Usage of Reparo:
     用于指定开始恢复的时间点，格式为 “2006-01-02 15:04:05”。如果不设置该参数则从最早的 binlog 文件开始恢复。
 -stop-datetime string
     用于指定结束恢复的时间点，格式同上。如果不设置该参数则恢复到最后一个 binlog 文件。
+-safe-mode bool
+    指定是否开启 safe mode，开启后可支持反复同步。
```
Comment (yikeke) [inline v3.0/reference/tools/tidb-binlog/reparo.md]:
```suggestion
    指定是否开启安全模式，开启后可支持反复同步。
```

File: v3.0/reference/tools/tidb-binlog/reparo.md [hunk 6]
```diff
@@ -60,6 +63,9 @@ log-level = "info"
 # 下游服务类型。 取值为 print, mysql（默认值：print）。当值为 print 时，只做解析打印到标准输出，不执行 SQL；如果为 mysql，则需要在 [dest-db] 中配置 host、port、user、password 等信息。
 dest-type = "mysql"
 
+# 安全模式配置。取值为 true, false (默认值：false)。当值为 true 时，Reparo 会将 update 语句拆分为 delete + replace 语句。
```
Comment (yikeke) [inline v3.0/reference/tools/tidb-binlog/reparo.md]:
```suggestion
# 安全模式配置。取值为 true 或 false（默认值：false）。当值为 true 时，Reparo 会将 update 语句拆分为 delete + replace 语句。
```

**Pull request merged**